### PR TITLE
feat: updated info to reflect Stacks 2.0 ecosystem

### PR DESCRIPTION
**Pull request by pgray-hiro**
Initial content for an Awesome Stacks page for Stacks 2.0.  This is meant to be a base seed that the community can start contributing to.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,53 +1,97 @@
...
+- **[Xverse Wallet](https://www.secretkeylabs.com/):** Mobile wallet application for managing STX and stacking
+
+### Web Applications
+
+- **[Stacks Explorer](https://explorer.stacks.co/?chain=mainnet):** An application for reviewing transactions on the Stacks blockchain
+- **[search-clarity.dev](https://search-clarity.dev):** An application for searching for deployed Clarity contracts
+- **[Pravica](https://pravica.io/):** Communication suite using Stacks identity services for secure messaging
+- **[Speed Spend](https://speed-spend.netlify.app):** A suite of working Clarity experiments on testnet ([source](https://github.com/friedger/speed-spend))
+
+## Clarity Resources
```
Comment (friedger) [inline README.md:24]: Maybe better ## Awesome Clarity Resources

**Review by friedger — CHANGES_REQUESTED** [review]
The updates looks quite Hiro biased :-) Maybe we just update the structure and then add awesome content

File: README.md [hunk 2]
```diff
@@ -1,53 +1,97 @@
...
-* **[App Center](https://app-center.openintents.org/):** [Open source](https://gitlab.com/friedger/app-center) app with app reviews and user comments.
+## Mining
 
-## Libraries
-### Authentication + Storage
-* **[`blockstack.js`](https://github.com/blockstack/blockstack.js):** The official JavaScript library for building apps with Blockstack with Typescript support
-* **[`react-blockstack`](https://github.com/njordhov/react-blockstack)**: React Hooks for Blockstack
-* **[`blockstack.go`](https://github.com/jackzampolin/blockstack.go):** A high-performance Go library for working with Blockstack
-* **[Blockstack Pascal](https://github.com/blackholeorganization/BlockstackPascal):** Library for Pascal
+- **[Daemon Technologies](https://daemontechnologies.co):** Mining bot for STX mining
```
[the remark this answers is not in the document]
Comment (pgray-hiro) [reply README.md:50]: Do you think we should remove this entirely for now? Or maybe move the entry lower in the document to a `## Companies` section?

**Comment by pgray-hiro** [comment]
@friedger Let me know what you think of the changes I made. I'm still learning the ecosystem so if I'm missing something it's probably because I don't know about it rather than an intentional omission.

File: README.md [hunk 3]
```diff
@@ -1,53 +1,105 @@
...
-* **[GaiaDown](https://github.com/AcidLeroy/gaiadown-ts):** Gaia storage backend for [LevelUP](https://github.com/Level/levelup).
+### Storage
 
+- **[CompassDB](https://github.com/eder-ai/compass-db):** A library for managing advanced collections in Gaia storage
+- **[lens-file-source](https://gitlab.com/MyLens/lens-file-source):** File storage abstraction through mechanisms like caching, transactions, and JSON map aggregation for Gaia
+- **[GaiaDown](https://github.com/AcidLeroy/gaiadown-ts):** Gaia storage backend for [LevelUp](https://github.com/Level/levelup)
+
+## NFTs
+
+- **[This is #1](https://www.thisisnumberone.com):** The first NFT built on Bitcoin and the Stacks blockchain ([contract](https://explorer.stacks.co/txid/SP3QSAJQ4EA8WXEDSRRKMZZ29NH91VZ6C5X88FGZQ.thisisnumberone-v2?chain=mainnet))
```
Comment (friedger) [inline README.md:66]: Not true, there are NFTs that exists before that.

File: README.md [hunk 4]
```diff
@@ -1,53 +1,105 @@
...
+- **[Official Stacks Forum](https://forum.stacks.org/):** Stacks community forum
+- **[r/stacks](https://www.reddit.com/r/stacks):** Stacks subreddit
+
+## Documentation
+
+- **[Official Stacks documentation](https://docs.stacks.co/):** Documentation and developer tutorials for learning
+  Clarity and developing Stacks apps
+- **[Stacks 101](https://stacks101.com):** Community curated STX knowledge
+
+## Companies
```
Comment (friedger) [inline README.md:91]: Does it make sense to list companies?

**Review by friedger — APPROVED** [review]
Let's merge it and continue from here.

**Pull request merged**